app/controllers/event_participations_controller.rb

```diff
@@ -0,0 +1,29 @@
+class EventParticipationsController < ApplicationController
+  include ApplicationHelper
```

Comment:
You don't seem to use it for anything here and those are (at least should) be intended for views.